runbot_travis2docker/models/runbot_repo.py

```diff
@@ -15,11 +15,18 @@ class RunbotRepo(models.Model):
 
     is_travis2docker_build = fields.Boolean('Travis to docker build')
     travis2docker_test_disable = fields.Boolean('Test Disable?')
+    travis2docker_image = fields.Char(
+        default=lambda s: s._default_travis2docker_image(),
+    )
     weblate_url = fields.Char(default="https://weblate.odoo-community.org/api")
     weblate_token = fields.Char()
     weblate_languages = fields.Char(help="List of code iso of languages E.g."
                                     " en_US,es_ES")
 
+    @api.model
+    def _default_travis2docker_image(self):
+        return 'vauxoo/odoo-80-image-shippable-auto'
```

Comment:
Yeah right now this is obscured deep in [Travis2Docker](https://github.com/Vauxoo/travis2docker/blob/0622bd064cd277791bb9997858130799869331ff/src/travis2docker/travis2docker.py#L69), so I'm just bringing it up a layer.